microservices/realtime/configs/config.yml

```diff
@@ -0,0 +1,20 @@
+ws:
+   listen: 127.0.0.1:8081
+   cert: configs/server.crt
+   key: configs/server.key
+   jwt_public: /opt/irontec/ivozprovider/storage/jwt/public.pem
+   jwt_private: /opt/irontec/ivozprovider/storage/jwt/private.pem
+redis:
+   master: mymaster
```

Comment:
Shouldn't this be resolved by sentinel?

Reply:
Master name is `mymaster` by default, so propably can be skipped and the library will assume that value.